pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[project]
+name = "package-releases-job"
+description = "Check packages updates on Python package indexes"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = {file = "LICENSE"}
+dynamic = ["version"]
+
+[project.scripts]
+package-releases-job = "producer:main"
+
+[build-system]
+requires = ["setuptools", "wheel", "setuptools_scm"]
+
+[tool.setuptools_scm]
```

Comment:
This might not be needed for this component, as this is not released  as module, just as a container image on release.